spec/models/contact_spec.rb

```diff
@@ -1,5 +0,0 @@
-require 'rails_helper'
-
-RSpec.describe Contact, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
-end
```

Comment:
This should stay as a unit test for the future when you have required fields (which there are some and should be added as validates_presence_of and then tested here.